### ISSUE TITLE
When trying to install an specific version it is not found datamaps@0.5.0

### ISSUE THREAD

**Issue by ramonmata** [markmarkoh/datamaps#287]
Hello there!

I was trying to install the latest version tags for the **datamaps** library and I am getting some errors with:

`npm install datamaps@0.5.1`

but instead I got the following errors:

```
npm install datamaps@0.5.1
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\rm\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "datamaps@0.5.1"
npm ERR! node v4.4.0
npm ERR! npm  v3.8.2

npm ERR! No compatible version found: datamaps@0.5.1
npm ERR! Valid install targets:
npm ERR! 0.4.4, 0.4.2, 0.4.0, 0.3.6
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

best regards!

**Comment by KurganskySergey** [comment]
When are you going to upload 0.5.x version of your lib to npm ?

**Comment by markmarkoh** [comment]
@ramonmata @KurganskySergey published v0.5.1

**Comment by KurganskySergey** [comment]
thnks!
